src/Symfony/Bridge/Monolog/Processor/WebProcessor.php

```diff
@@ -22,10 +22,13 @@
  */
 class WebProcessor extends BaseWebProcessor
 {
-    public function __construct()
+    /**
+     * @param array|null $extraFields
+     */
```

Comment:
This doesn't add any value. You should either remove it or provide a reasonable description.